dms/models/dms_security_mixin.py

```diff
@@ -237,6 +237,8 @@ def check_access_groups(self, operation):
             return None
         group_ids = set(self.ids) - set(self._get_ids_without_access_groups(operation))
         if group_ids:
+            # Hack to prevent error in some cases (unlink subdirectory) re-compute
+            self.complete_group_ids
```

Comment:
Please explain the problem and why is this a hack. I think more than this triggers the recomputation of that, but this denotes a bug in the way things are computed.

Reply:
Now, it's not necessary, sorry